src/app/pages/events/sfd-2024/components/hero/hero.component.html

```diff
@@ -4,8 +4,10 @@
     <h1
       class="text-5xl md:text-6xl font-bold leading-tighter tracking-tighter my-4 font-heading dark:text-gray-200">
       Celebremos el
-      <span>software libre</span> y el
-      <span class="text-accent highlight"> código abierto</span>
+      <span style="text-shadow: rgb(1, 1, 1) 1px 0 10px;">software libre</span>
+      y el
+      <span style="text-shadow: rgb(1, 1, 1) 1px 0 10px;"
+        class="text-accent highlight"> código abierto</span>
```

Comment:
Aplicar los estilos por clases tailwind o por [@apply](https://tailwindcss.com/docs/functions-and-directives#apply) vía scss

Reply:
Además, dejar el shadow como estaba en dark mode, que este text-shadow solo se aplique en light mode